pkg/query/storeset_test.go

```diff
@@ -27,7 +27,12 @@ var testGRPCOpts = []grpc.DialOption{
 }
 
 type testStore struct {
-	info storepb.InfoResponse
+	info      storepb.InfoResponse
+	ruleGroup *storepb.RuleGroup
+}
+
+func (s *testStore) Rules(_ *storepb.RulesRequest, srv storepb.Rules_RulesServer) error {
+	return srv.Send(storepb.NewRuleGroupResponse(s.ruleGroup))
```

Comment:
Hm.. this is quite inconsistent. We don't test Series this way. I would either do Series the same as move rule group testing somewhere else (as it is with Series).

Most likely e2e is the correct place

Reply:
ah thanks, i really wasn't sure either if this is the right place.